zaza/openstack/charm_tests/ceph/rbd_mirror/tests.py

```diff
@@ -31,6 +31,7 @@
 
 
 DEFAULT_CINDER_RBD_MIRRORING_MODE = 'pool'
+INTERNAL_POOLS = set(('.mgr', 'device_health_metrics'))
```

Comment:
It's probably going to sound a bit mad, but this would be better as a tuple, rather than a set.  This is because, as a tuple, it would be an immutable collection, rather than a mutable one, which just removes a whole class of potential errors; as it's being used as a constant, and only as something to iterate over (e.g. used as an iterator), then this won't change the semantics of the code:

```python
INTERNAL_POOLS = (('.mgr', 'device_health_metrics'), )
```

Note the extra `,` above to ensure that it's a single item tuple.

Reply:
what about using a frozenset ? It would be a constant, and none of the code would need to change (it would still be usable by `set.difference`).

Reply:
yeah, I'll go with a `frozenset`. It's the closest to the current semantics while also being a constant and avoiding any potential pitfalls.